File: src/main/java/no/nav/syfo/domain/Oppgave.java
package no.nav.syfo.domain;

import lombok.Builder;
import lombok.Getter;
import lombok.Value;

import java.time.LocalDate;

@Value
@Getter
@Builder
public class Oppgave {
    String oppgaveId;
    int versjon;
    String beskrivelse;
    LocalDate aktivFra;
    LocalDate aktivTil;
    String oppgavetype;
    String fagomrade;
    String prioritet;
    String ansvarligEnhetId;
    String saksnummer;
    String dokumentId;
    String status;
    String geografiskTilknytning;
}

File: src/main/kotlin/no/nav/syfo/domain/Oppgave.kt
package no.nav.syfo.domain

import java.time.LocalDate

data class Oppgave(
    val oppgaveId: String? = null,
    val versjon: Int = 0,
    val beskrivelse: String? = null,
    val aktivFra: LocalDate? = null,
    val aktivTil: LocalDate? = null,
    val oppgavetype: String? = null,
    val fagomrade: String? = null,
    val prioritet: String? = null,
    val ansvarligEnhetId: String? = null,
    val saksnummer: String? = null,
    val dokumentId: String? = null,
    val status: String? = null,
    val geografiskTilknytning: String? = null
)
